### ISSUE TITLE
Search pages: display "no results" in the case of no results

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#575]
@dchiller made a good suggestion:

> When I search for something that doesn’t have any results, it might be nice to show a message that says something like “No chants found.” Right now, you press the “Apply” button and there are ways to see that something has happened (eg, the url changes), but it does look a little like “nothing happened” and I think explicitly showing that a search was run and returned no results could be good so that someone doesn’t, for example, think the button is broken if they aren’t looking too closely for other signs of activity.
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/58090591/222761343-b88b7368-6630-46d3-9351-608741be944d.png">

**Comment by jacobdgm** [comment]
We've recently implemented this on a number of pages. @lucasmarchd01 - when you get a chance, can you confirm that we've applied this in all the places it makes sense (and then close the issue if you find this to be true)?
